policy/modules/admin/tomoyo-auditd.if

```diff
@@ -0,0 +1,27 @@
+## <summary>Audit log daemon for TOMOYO Linux MAC.</summary>
+
+########################################
+## <summary>
+##	All of the rules required to
+##	administrate tomoyo-auditd.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`tomoyo_auditd_admin',`
+	gen_require(`
+		type tomoyo_auditd_t;
+		type tomoyo_auditd_etc_t;
+		type tomoyo_auditd_log_t;
+	')
+
+	admin_pattern($1, { tomoyo_auditd_etc_t })
```

Comment:
no need for the {}, theres nothing to expand

Reply:
ok, thanks